docs/release-source/release/sandbox.md

```diff
@@ -26,23 +24,26 @@ describe('myAPI.hello method', function () {
 
     it('should be called once', function () {
         myAPI.hello();
-        sinon.assert.calledOnce(myAPI.hello);
+        sandbox.assert.calledOnce(myAPI.hello);
     });
 
     it('should be called twice', function () {
         myAPI.hello();
         myAPI.hello();
-        sinon.assert.calledTwice(myAPI.hello);
+        sandbox.assert.calledTwice(myAPI.hello);
     });
 });
 ```
 
 ## Sandbox API
 
-#### `var sandbox = sinon.createSandbox();`
+#### Default sandbox
 
-Creates a sandbox object with spies, stubs, and mocks.
+Since `sinon@5.0.0`, the `sinon` object is a default sandbox. Unless you have a very advanced setup or need a special configuration, you probably want to just use that one.
```

Comment:
I think I'd also add a line here explaining why this change happened and how this makes testing easier, especially mentioning how useful it is to clear/restore all methods at once without having to create lots of sandboxes on each test.

Reply:
At first I agreed with you, and wrote some documentation.

But then I thought about it some more. The motivation is only really interesting when you're transitioning from earlier versions to this MAJOR version. Once you're over that hump, stating that in the documentation just adds noise.

Instead, I've put details into the migration guide for v5, and have pushed a new commit with that.